pkg/log/loglevels.go

```diff
@@ -0,0 +1,8 @@
+package log
+
+const (
+	TRACE_LEVEL   = 100
+	DEBUG_LEVEL   = 10
+	VERBOSE_LEVEL = 4
+	INFO_LEVEL    = 0
```

Comment:
Sorry I think I got it wrong, and default should be 1?

Reply:
Probably we should remove "INFO_LEVEL" altogether and then, use
klog.Info( ... ) for "always-on" logs.